src/clique_json_writer.erl

```diff
@@ -53,8 +53,10 @@
 
 -spec write(status()) -> {iolist(), iolist()}.
 write(Status) ->
-    PreparedOutput = lists:reverse(prepare(Status)),
-    {[mochijson2:encode(PreparedOutput), "\n"], []}.
+    case application:get_env(clique, json_encoder, undefined) of
+        undefined -> throw({error, undefined_json_encoder});
+        {M, F} -> {[M:F(prepare(Status)), "\n"], []}
```

Comment:
is the `lists:reverse/1` call specific to mochijson2 or are we going to miss this?

Reply:
It shouldn't matter even though the generated JSON is a list of JSON objects. I don't think the output ordering is very important and usually it's a list of a single element.